tsfc/fem.py

```diff
@@ -153,12 +164,17 @@ def basis_evaluation(self, finat_element, local_derivatives, entity_id):
 
 
 class Translator(MultiFunction, ModifiedTerminalMixin, ufl2gem.Mixin):
-    """Contains all the context necessary to translate UFL into GEM."""
+    """Multifunction for translating UFL -> GEM.  Incorporates ufl2gem.Mixin, and
+    dispatches on terminal type when reaching modified terminals."""
 
     def __init__(self, context):
+        # MultiFunction.__init__ does not call further __init__
+        # methods, but ufl2gem.Mixin must be initialised.
+        # (ModifiedTerminalMixin requires no initialisation.)
```

Comment:
I guess this is working around `MultiFunction.__init__` not calling super?

Reply:
Well, yes, we have to do it the non-`super` way, calling it for each base class manually.